tests/array/tarraylen.nim

```diff
@@ -0,0 +1,33 @@
+discard """
+  file: "tarraylen.nim"
+  output: '''0
+1
+1
+1
+1
+2
+2
+3
+3
+3
+3
+1
+3
+1
+'''
+"""
+var a: array[0, int]
```

Comment:
oh come on. why not just `assert`s? How can you read which output line corresponds to which `echo`?